src/cargo/ops/cargo_add/mod.rs

```diff
@@ -565,7 +566,7 @@ fn populate_dependency(mut dependency: Dependency, arg: &DepOp) -> Dependency {
     }
 
     if let Some(rename) = &arg.rename {
-        dependency = dependency.set_rename(rename);
+        dependency = dependency.set_rename(rename).unwrap();
```

Comment:
If you have a reason to believe an `unwrap` is justified, it should be documented, ideally as an `except`.  Otherwise, chain the error